examples/tokens/readme.md

```diff
@@ -0,0 +1,90 @@
+# Token Transfer Example
+
+This is an example of using the SDK to mint, and send tokens from one ChainTree to another.
+
+The example program will use the public Tupelo testnet, so you will need to be connected to the
+Internet.
+
+There's a video walkthrough of token transfer here:
+[![Play video](http://img.youtube.com/vi/kjysUm0eTKc/1.jpg)](https://youtu.be/kjysUm0eTKc "Wallet Walkthrough")
+
+Unlike other systems, Tupelo models individual objects *first* (as ChainTrees) and then layers tokens on top of those individual objects.
+
+ChainTrees may mint their own tokens and send and receive tokens from other ChainTrees. Within a ChainTree a token can have an arbitrary name, the canonical name of the token is the DID of the ChainTree prepended on that arbitrary name. For example if ChainTree `did:tupelo:xyz` minted a `stockToken` the canonical token name would be `did:tupelo:xyz:stockToken`
+
+The basic flow of creating and sending/receiving tokens is as follows:
+
+1. Establish Token
+1. Mint Token
```

Comment:
I don't know if this idiomatic in Markdown, but isn't it more readable to use real item numbers instead, so the Markdown source reads better? No biggie, just wondering, since I personally write the item numbers out.

Reply:
Starting each item in an ordered list with "1." in the source allows you to add, remove, and reorder items while editing the document without having to worry about keeping track of the numbers. The markdown formatting engine puts the correct numbers in automatically in the published document.

Reply:
@zonotope Yes, I'm aware how it works. I was more thinking about the experience of reading the Markdown source, but I respect the choice of making editing easier.